pom.xml

```diff
@@ -8,7 +8,7 @@
   </parent>
   <groupId>us.bpsm</groupId>
   <artifactId>edn-java</artifactId>
-  <version>0.5.0</version>
+  <version>0.6.0-SNAPSHOT</version>
```

Comment:
I think you'll find that the master branch only contains release versions and the develop branch builds as snapshot. (This repo attempts to follow the git-flow branching model.)